hax/main.js

```diff
@@ -0,0 +1,147 @@
+(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);throw new Error("Cannot find module '"+o+"'")}var f=n[o]={exports:{}};t[o][0].call(f.exports,function(e){var n=t[o][1][e];return s(n?n:e)},f,f.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
+'use strict';
+
+var App = require('./stuff/App');
+var Blinker = require('./stuff/Blinker')
+
+
+var Main = (function() {
+
+	return {
+
+		initialize: function() {
+			this._blinkingStuff = [];
+
+			this._initApp();
+			this._initBlinkers();
+			console.log('oh shit whadup');
+			//var pallies = document.getElementById('');
+		},
+		_initApp: function() {
+			var canvas = document.getElementById('it');
+			var myApp = new App(canvas);
+			myApp.draw();
+		},
+		_initBlinkers: function() {
+			this._blinkingElements = document.querySelectorAll('[data-blink]');
+			var myLength = this._blinkingElements.length;
+			for (var i = 0; i < myLength; i++) {
+				var element = this._blinkingElements[i];
+				var optionsData = element.getAttribute('data-blink');
+				var options = JSON.parse(optionsData);
+				var myBlinky = new Blinker(element, options);
+				this._blinkingStuff.push(myBlinky);
+			}
+		}
+
+
+	};
+
+}());
+
+module.exports = Main.initialize();
+
+},{"./stuff/App":2,"./stuff/Blinker":3}],2:[function(require,module,exports){
+'use strict';
+
+function App(element){
+	this._canvas = element;
+	this._width = this._canvas.width;
+	this._height = this._canvas.height;
+	this._context = this._canvas.getContext('2d');
+
+	this._NUM_RECTS = 10;
+	this._DX = this.width / this._NUM_RECTS;
+	this._DY = this.height / this._NUM_RECTS;
+
+	this._setup();
+}
+
+var proto = App.prototype;
+
+proto._setup = function(){
+	this._context.strokeStyle = "rgba(0, 0, 0, .1)";
+	this._context.lineWidth = "0.2";
+	this._context.lineWidth = "0.1";
+}
+
+proto.update = function() {
+
+}
+
+proto._drawBox = function(x, y ,w, h) {
+	var top = x - w/2.;
+	var bot = y - h/2.;
+	this._context.rect(top, bot, w, h);
+	this._context.stroke();
+}
```

Comment:
can we just track the `min` files then?

On Fri, Jul 8, 2016 at 9:17 PM, greg elwood <notifications@github.com>
wrote:

> In hax/main.js
> <https://github.com/internet-research-labs/soakinthatlane.business/pull/2#discussion_r70159504>
> :
>
> > +proto._setup = function(){
> > +	this._context.strokeStyle = "rgba(0, 0, 0, .1)";
> > +	this._context.lineWidth = "0.2";
> > +	this._context.lineWidth = "0.1";
> > +}
> > +
> > +proto.update = function() {
> > +
> > +}
> > +
> > +proto._drawBox = function(x, y ,w, h) {
> > +	var top = x - w/2.;
> > +	var bot = y - h/2.;
> > +	this._context.rect(top, bot, w, h);
> > +	this._context.stroke();
> > +}
>
> also a built file... the files are in /src/
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/internet-research-labs/soakinthatlane.business/pull/2/files/2365d39822e8fab2fccbc398ea2c961da1549b54#r70159504>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAWcrWsbJX-xcRV-Gf-pbsgVNvozI7oOks5qTvargaJpZM4JId_5>
> .
>